iati/core/tests/test_resources.py

```diff
@@ -158,6 +155,20 @@ def test_load_as_string(self):
         assert isinstance(result, six.string_types)
         assert result == 'This is a string that is not valid XML\n'
 
+    @pytest.mark.parametrize("load_method", [iati.core.resources.load_as_bytes, iati.core.resources.load_as_dataset, iati.core.resources.load_as_string])
+    def test_load_as_x_non_existing_file(self, load_method):
+        """Test that resources.load_as_bytes returns a bytes object with the expected content."""
```

Comment:
backticks? I'm still not clear on which things need them and which don't =/

Reply:
References to other pieces of code should have them.